.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
 max-line-length=100
-ignore: E252, E301, E302, E305, E731, F811
+ignore: E252, E301, E302, E305, W503, W605, E731, F811
```

Comment:
W503 & W504 are mutually exclusive, one must be ignored: 
W503: binary operator must come before break
W504: binary operator must come after break. 

pep8 prefers W504, so we ignore W503

W605 breaks on the starfish art. 

E252 complains if you don't have spaces around equals sign in parameter kwargs.

Reply:
If W605 breaks only on the starfish art, let's just fix that.

Reply:
Also, my guess is that the warnings in this file are not useful any more, since we specify them on the command line.

Reply:
#780